### ISSUE TITLE
Fixing Labels for New Data

### ISSUE THREAD

**Issue by Tunguyen485** [MaartenGr/BERTopic#991]
Hi Maarten,
We try to fix the labels in a certain period and force the new data to the labels that we currently have (not introduce the new label until the next run), but we do not find the best way to do it yet. 
Can you help us fix the label of the new data whenever we run BERTopic in a certain period of time?
The model that we used is below. In this case, we use KMeans to force the outliers to certain topics, and remove stopwords instead of using UMAP.   

sentence_model = SentenceTransformer("all-mpnet-base-v2")
embeddings = sentence_model.encode(docs, show_progress_bar=True)

cluster_model = KMeans(n_clusters =n,  init='k-means++', max_iter=600, n_init=80, random_state=42)
vectorizer_model = CountVectorizer(ngram_range=(1, 3), stop_words="english")

topic_model = BERTopic(hdbscan_model=cluster_model, 
                       vectorizer_model=vectorizer_model,
                       language = 'english',
                      calculate_probabilities = True,
                      verbose=True)

topics, probabilities = topic_model.fit_transform(docs, embeddings)

**Comment by MaartenGr** [comment]
When you want to predict the labels of unseen documents, documents you did not train on, you would only need to use the `.transform` function. It will not update the internal model and it will force new data to the labels that you currently have.

**Comment by Tunguyen485** [comment]
Thank you, Maarten.
